src/components/CaseDetails/styles.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
+
+// case detail
+export const CaseDisplay = styled.aside`
+  position: relative;
+  width: 100%;
+`;
+
+export const CaseTitle = styled.h1`
```

Comment:
David changed the fonts to Open Sans. I think you want this component to inherit from H1 (a styled h1 defined in `@/styles/text` which has the right size + font + color.

Reply:
updated! 🥳